### ISSUE TITLE
Custom timestamps?

### ISSUE THREAD

**Issue by MilanObrenovic** [Danktuary/react-discord-message#28]
Is it possible to change the timestamp of posted message? Instead of automatically displaying current date, maybe have a configuration option to display it as current timestamp of hh:mm:ss or let us manually write time?

Examples of actual discord formatting:
Today at 7:12 PM
Yesterday at 1:15 PM
01/25/2022

**Comment by milanobrenovic** [comment]
> Is it possible to change the timestamp of posted message? Instead of automatically displaying current date, maybe have a configuration option to display it as current timestamp of hh:mm:ss or let us manually write time?

Examples of actual discord formatting:
Today at 7:12 PM
Yesterday at 1:15 PM
01/25/2022

**Comment by Danktuary** [comment]
Both the `DiscordEmbed` and `DiscordMessage` components support a `timestamp` prop that accepts either a Date object or a valid value to be passed in `new Date()`, e.g. `'01/02/2022'` as a string. Timestamps will always be displayed in `mm/dd/yyyy` format.

**Comment by MilanObrenovic** [comment]
> Timestamps will always be displayed in `mm/dd/yyyy` format.

Yes that's why i asked this question, is it possible to display "Today at 12:36 AM" instead of "01/29/2022" ?

**Comment by Danktuary** [comment]
No it's currently not possible, I don't have any plans to update it as such

**Comment by MilanObrenovic** [comment]
> No it's currently not possible, I don't have any plans to update it as such

Could you please make this one change if you have some spare time? Ability for manually setting custom timestamp with hours and minutes can be important for visual representation. I'd make a pull request to update this but i work in angular and not react
